src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassReportServiceImpl.java

```diff
@@ -284,6 +284,19 @@ private void writeHeaderForResultsDetailsByResultCode(Class businessObjectClass,
 
         reportWriterService.writeFormattedMessageLine(reportMessage);
     }
+    
+    private String buildErrorLines(String prefix, String errorMessage) {
+        StringBuffer errorLines = new StringBuffer();
+        String[] lines = errorMessage.split(KFSConstants.NEWLINE);
+        for (String line : lines) {
+            if (StringUtils.isNoneBlank(line)) {
```

Comment:
Since you're only checking a single String with this call, I think it would be better to use an isNotBlank() check instead of an isNoneBlank() check.

Reply:
Thank you Chad. I have changed it.